conda-installation.md

```diff
@@ -137,29 +155,39 @@ When this step finishes, you should see a file in your current directory named
 The job will need to go through a few steps to use this "packed" conda environment; 
 first, setting the `PATH`, then unzipping the environment, then activating it, 
 and finally running whatever program you like. The script below is an example 
-of what is needed - change the `env-name` to the name of your environment. 
+of what is needed (customize as indicated to match your choices above).
 
 ```
 #!/bin/bash
 
+set -e
+
+# replace env-name on the right hand side of this line with the name of your conda environment
+env_name=env-name
```

Comment:
I *almost* did this! Not everyone knows how bash variables work, so I'm on the fence about it. If we go the variable route, I would recommend:
```
ENVNAME=env-name
ENVDIR=$ENVNAME
```
at the start, and then proceed with the script.

Reply:
I'm torn... the programmer in me hates that the user has to change the name in multiple places (I, of course, missed one of the places the first time I tried the tutorial...), but doing it this way also makes this part ~mysterious~, when it's really pretty straightforward. I defer to your wisdom! Let me know which you prefer and I will make it so.

Reply:
The RCF team endorses variables. :)